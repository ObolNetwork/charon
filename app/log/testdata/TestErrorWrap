00:00 [31mERRO[0m err1: first {"1": 1, "caller": "log_test.go:59"}
	app/log/log_test.go:54 .TestErrorWrap
00:00 [31mERRO[0m err2: second: first {"2": 2, "1": 1, "caller": "log_test.go:60"}
	app/log/log_test.go:54 .TestErrorWrap
00:00 [31mERRO[0m err3: third: second: first {"3": 3, "2": 2, "1": 1, "caller": "log_test.go:61"}
	app/log/log_test.go:54 .TestErrorWrap
