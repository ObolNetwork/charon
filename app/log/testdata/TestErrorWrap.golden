00:00 [31mERRO[0m err1: first {"1": 1, "caller": "log_test.go:61"}
	app/log/log_test.go:56 .TestErrorWrap
00:00 [31mERRO[0m err2: second: first {"2": 2, "1": 1, "caller": "log_test.go:62"}
	app/log/log_test.go:56 .TestErrorWrap
00:00 [31mERRO[0m err3: third: second: first {"3": 3, "2": 2, "1": 1, "caller": "log_test.go:63"}
	app/log/log_test.go:56 .TestErrorWrap
